package.json

```diff
@@ -23,13 +23,11 @@
   },
   "homepage": "https://github.com/mmarkelov/jest-playwright#readme",
   "scripts": {
-    "build": "rm -rf lib/ && babel src -d lib --ignore '**/*.test.js'",
+    "build": "tsc",
     "format": "prettier --write \"*.{js,md}\" \"src/**/*.js\"",
     "lint": "eslint .",
     "prepublishOnly": "yarn run build",
-    "test:bin": "jest src/bin/utils.test.js",
```

Comment:
I have not understood why you splitted them up there. So for each a new Jest instance would launch which would result in a longer test run.